### DISCUSSION TITLE
"Packages" pane is missed

### DISCUSSION THREAD

**Discussion by seewe** [posit-dev/positron#5841] [Feedback] [answered]
Positron is a very great idea of IDE ! I realy enjoy it !
However, i found that the actual version of the IDE miss the "Packages" pane which list all packages in the system or project. It may be great to have the same thing than the one in Rstudio IDE. it's more practical to get it and access each package documentation through that.

![image](https://github.com/user-attachments/assets/bdb171b8-7bfc-49dc-8357-d8736d3f0c82)

**Comment by juliasilge** [comment, marked as answer]
We have a couple of issues on this topic that you may want to take a look at:

- https://github.com/posit-dev/positron/issues/1369
- https://github.com/posit-dev/positron/issues/1391

Please feel free to 👍 anything that resonates with your use case, or add additional details!

**Comment by kv9898** [comment]
For anyone who is interested, I have made the [Positron R Package Manager](https://github.com/kv9898/positron-r-package-manager) extension for this feature!

<img src="https://github.com/user-attachments/assets/68ae31ca-0e6d-4c90-a783-132408dee896" alt="Positron R Package Sidebar" width="300"/>

You can access it from the [Open VSX marketplace](https://open-vsx.org/extension/kv9898/positron-r-package-manager):

<img src="https://github.com/user-attachments/assets/7fd8d6c7-2e20-4bbf-b481-91e3bf271dc1" alt="Open VSX page" width="500"/>

**Comment by ntluong95** [comment]
@kv9898 Use your code template, I was able to reproduce for Python, but still need more effort to polish it since Python package manager is much more complicated than R.
![image](https://github.com/user-attachments/assets/8a29afb7-9d10-4ee9-9b75-a32cc07ee349)

**Reply by kv9898** [reply]
> @kv9898 Use your code template, I was able to reproduce for Python, but still need more effort to polish it since Python package manager is much more complicated than R.
> ![image](https://github.com/user-attachments/assets/8a29afb7-9d10-4ee9-9b75-a32cc07ee349)
> 

This looks fantastic, as EXACTLY how I had imagined it! Can't wait to try it out! (As a uv user I would love to see the ability to use uv for global package installations)

**Comment by seewe** [comment]
Great job guys ! I realy enjoy this !
